js/src/framework.ts

```diff
@@ -118,12 +136,13 @@ export async function Eval<Input, Output, Expected>(
           progressReporter,
           []
         );
-        reportEvaluatorResult(name, ret, true);
+        reportEvaluatorResult(name, ret, true, false);
```

Comment:
Can we change these to named arguments? Or if backwards-compatibility is a must, add any new arguments as named arguments?

Reply:
fixed